ground/src/main/java/com/google/android/ground/ui/map/gms/mog/MogTileReader.kt

```diff
@@ -27,6 +27,7 @@ class MogTileReader(private val inputStream: InputStream, initialOffset: Long) {
   private var offset: Long = initialOffset
 
   fun readTiles(tiles: List<MogTileMetadata>): Flow<MogTile> = flow {
+    // Tiles must be read sequentially since input stream is shared among all.
```

Comment:
"input stream is shared among all" what? Not sure what this is referring to.

Reply:
Updated the comment